commands/inventory/add_agent_qan_mysql_slowlog_agent.go

```diff
@@ -93,4 +115,6 @@ func init() {
 	AddAgentQANMySQLSlowlogAgentC.Flag("password", "MySQL password for scraping metrics").StringVar(&AddAgentQANMySQLSlowlogAgent.Password)
 	AddAgentQANMySQLSlowlogAgentC.Flag("custom-labels", "Custom user-assigned labels").StringVar(&AddAgentQANMySQLSlowlogAgent.CustomLabels)
 	AddAgentQANMySQLSlowlogAgentC.Flag("skip-connection-check", "Skip connection check").BoolVar(&AddAgentQANMySQLSlowlogAgent.SkipConnectionCheck)
+	AddAgentQANMySQLSlowlogAgentC.Flag("disable-queryexamples", "Disable collection of query examples").BoolVar(&AddAgentQANMySQLSlowlogAgent.DisableQueryExamples)
+	AddAgentQANMySQLSlowlogAgentC.Flag("size-slow-logs", "Rotate slow log file at this size (0 disables rotation)").BytesVar(&AddAgentQANMySQLSlowlogAgent.MaxSlowlogFileSize)
```

Comment:
according what I've seen in pmm-managed 0 sets default value to 1GB
https://github.com/percona/pmm-managed/pull/264/files#diff-ce82fa739acbb612f2b2db458c067721R48

Reply:
Inventory API is dump – 0 or absence of value means 0.
Management API is smart – 0 or absence of value means "safe default" (1 GB).

This is `pmm-admin inventory` command. Linked pmm-managed code is Management API.